components/multiplayer/multiplayerService.ts

```diff
@@ -174,14 +174,14 @@ multiplayerRouter.post(
             // join existing match
             const match = activeMatches.get(req.body.matchId)!
 
-            match.Players.forEach((playerId) =>
+            for (const playerId of match.Players) {
```

Comment:
Is there a particular reason you did this, and if so, should we do it everywhere where forEach is used (about 26 times)? Should be added to convention list and potentially ban using eslint.

Reply:
Foreach looks better imo